test/index.test.js

```diff
@@ -154,4 +162,43 @@ describe('GoodInflux', () => {
         }).to.throw(Error, 'Unsupported protocol ftp:. Supported protocols are udp, http or https');
         done();
     });
+
+    it('Ommit metadata when not defined', (done) => {
```

Comment:
Nice work adding tests for these two cases.  *Minor nit:* "Omit" is spelled with one m.  ;-)